web/satellite/src/components/common/VButton.vue

```diff
@@ -11,21 +11,36 @@
         <slot name="icon" />
         <div v-if="isWhiteGreen" class="greenCheck">&#x2713;</div>
         <div v-if="isGreenWhite" class="whiteCheck">&#x2713;</div>
-        <div v-if="hasTrashIcon" class="trash-icon"><TrashIcon /></div>
-        <span class="label" :class="{uppercase: isUppercase}">{{ label }}</span>
+        <span class="label" :class="{uppercase: isUppercase}">
+            <LockIcon v-if="icon.toLowerCase() === 'lock'" />
+            <CreditCardIcon v-if="icon.toLowerCase() === 'credit-card'" />
+            <DocumentIcon v-if="icon.toLowerCase() === 'document'" />
+            <TrashIcon v-if="icon.toLowerCase() === 'trash'"/>
+            <span v-if="icon !== 'none'">&nbsp;&nbsp;</span>{{ label }}</span>
```

Comment:
is `&nbsp;&nbsp;` intentional?

Reply:
Yes, It was used as a quick way to manually add space if the user decides to use an icon.